controllers/members.js

```diff
@@ -14,7 +14,7 @@ const ERROR_MESSAGE = "Something went wrong. Please try again or contact admin";
 
 const getMembers = async (req, res) => {
   try {
-    const allUsers = await members.fetchUsers();
+    const allUsers = await members.fetchUsers(req.query);
```

Comment:
Is this XSS-safe?

Reply:
I think this is safe from XSS since the value of the query param won't be going in the response and further to the HTML output.
Also if there were some other query parameter also being used for this api endpoint e.g. private which we think can be misused by some user then we need to restrict the it's usage by setting some role based authorisation.

Reply:
Added validation middleware to validate the query params for the /members endpoint